### PR TITLE
[Zoe] Part 5 of Switching to Zoe Invites (Covered Call)

### PR DESCRIPTION
**Pull request by katelynsills**
This PR makes the necessary changes to `coveredCall.js` and its tests.

### CODE REVIEW

**Review by erights — CHANGES_REQUESTED** [review]
I know we're not up to Part 5 yet, but here are my reactions so far.

File: packages/zoe/contracts/coveredCall.js [hunk 1]
```diff
@@ -1,77 +1,68 @@
...
+    zoe,
+    terms.assays,
+  );
+  let sellerHandle;
 
-      const customInviteExtent = {
-        offerMadeRules: offerRules,
-        offerToBeMade: makeExactlyMatchingPayoutRules(payoutRules),
-      };
+  const makeCallOptionInvite = () => {
```
[the remark this answers is not in the document]
Comment (katelynsills) [reply packages/zoe/contracts/coveredCall.js]: Good idea

File: packages/zoe/test/unitTests/contracts/test-coveredCall.js [hunk 2]
```diff
@@ -10,185 +10,150 @@ import { setup } from '../setupBasicMints';
...
-    // 7: Bob does a claimAll on the escrowReceipt payment
-    const bobEscrowReceipt = await escrowReceiptAssay.claimAll(
-      allegedBobEscrowReceipt,
+    const { seat: bobSeat, payout: bobPayoutP } = await zoe.redeem(
+      bobExclOption,
+      bobOfferRules,
+      bobPayments,
     );
 
     // 8: Bob makes an offer with his escrow receipt
```
Comment (erights) [inline packages/zoe/test/unitTests/contracts/test-coveredCall.js]: What does it mean that this testcase has steps 6,8,13,14 but no steps 7,9,10,11,12 ?

Comment (katelynsills) [reply packages/zoe/test/unitTests/contracts/test-coveredCall.js]: It means that trying to number the steps of a test when the test changes drastically from week to week was a bad idea. :) I'll remove the numbers.

Comment (katelynsills) [reply packages/zoe/test/unitTests/contracts/test-coveredCall.js]: Ok, those numbered steps should be removed now

**Review by erights — APPROVED** [review]
Make a note somewhere about what work until https://github.com/Agoric/agoric-sdk/issues/368 is fixed.

LGTM. Full steam ahead!

**Comment by katelynsills** [comment]
Great! I just added a comment to #368 to explain what will work and won't work now.

**Pull request merged**